docs/src/main/paradox/openshift.md

```diff
@@ -7,6 +7,7 @@ Install the following:
 * [Docker](https://docs.docker.com/install/)
 * [Kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
 * OpenShift's CLI: [`oc`](https://docs.openshift.com/container-platform/3.10/cli_reference/get_started_cli.html#installing-the-cli) (["Installing the CLI"](https://docs.openshift.com/container-platform/3.10/cli_reference/get_started_cli.html#installing-the-cli))
+* [`kustomize`](https://github.com/kubernetes-sigs/kustomize) (v2.0.0+)
```

Comment:
I would be happy to not introduce another tool. If I understand it correctly, the only reason to have `kustomize` is to avoid duplications of yaml. 

I don't find a big deal to have this sort of duplication because those files are pretty much static. We surely have a little bit more of maintenance when we introduce duplication, but for the user, it becomes straightforward, for instance:

```
 kubectl apply -f deployment/kubernetes.yml
```
or 

```
 kubectl apply -f deployment/openshift.yml
```

Reply:
`kustomize` is not just nice for YML manipulation, but also for simple concatenations. Using `kustomize` we can split `deployment/kubernetes.yml` and `deployment/openshift.yml` into:

```
base/service.yml
base/deployment.yml
overlays/minikube/ingress.yml
overlays/minishift/route.yml
overlays/centralpark/route.yml
```

Which no only is more maintainable but also allows focusing on the small differences.

I think IRL, users may find value on knowing how to adapt their YML's.